firebase.json

```diff
@@ -1,7 +1,14 @@
 {
+  "database": {
+    "rules": "rules/database/database.rules.json"
```

Comment:
I see that Realtime database rules and tests have been introduced but I do question whether we should have these rules and tests since we are not using Realtime database in any of our clients. I think that having this in code makes the repo more complex (due to all files and tests to do) than it is and gives an impression that we're using Realtime database which we are not.

Reply:
true, I had considered that it would be useful to include the rules in our code to ensure that no one gave access through the console. By defining them here it is a single place for all the definitions.

Reply:
remove unused rules and tests